src/models/salt.js

```diff
@@ -29,6 +29,14 @@ export default function buildSalt (sequelize, DataTypes) {
     value: {
       type: DataTypes.STRING,
       allowNull: false
+    },
+    createdAt: {
+      type: DataTypes.DATE,
+      allowNull: false
+    },
+    updatedAt: {
+      type: DataTypes.DATE,
+      allowNull: false
```

Comment:
This field can be null. The semantic here is `null = "field hasn't changed"`.

Reply:
I've read Sequelize's documentation again and I think this definition is not even necessary in the Model, since this is an automatic field managed by the library.

Reply:
Yes. You're right. It's only necessary for your migration file.